src/encryption/encryption.go

```diff
@@ -103,3 +106,60 @@ func LogGroups(groups []Group, logger *zap.SugaredLogger) {
 		}
 	}
 }
+
+func (v *Value) encryptedPath() string {
+	return "#" + v.path.String()
+}
+
+func prepareMapToEncrypt(values []Value) map[string]string {
+	result := make(map[string]string)
+	for _, value := range values {
+		result[value.encryptedPath()] = value.value
+	}
+	return result
+}
+
+func DoEncrypt(projectState *state.State, unencryptedGroups []Group, api *Api) error {
+	errors := utils.NewMultiError()
+	localManager := projectState.LocalManager()
+	projectId := fmt.Sprintf("%v", projectState.Manifest().Project.Id)
+	for _, group := range unencryptedGroups {
+		var encryptionError error
+		// create map with {"#<value-path>":"<value-to-encrypt>"...} entries
+		mapToEncrypt := prepareMapToEncrypt(group.values)
+
+		//type switch on config or configRow state
+		switch o := group.object.(type) {
+		case *model.ConfigState:
+			encryptedMap, encryptionError := api.EncryptMapValues(o.ComponentId, projectId, mapToEncrypt)
```

Comment:
Podla mna tu na prehladnosti kodu dost ubera to, ze skoro kazda premenna je prefixovana s `encrypted`.

Reply:
skusil som to premenovat, ale nenapadlo ma nic lepsie, takto to nakoniec necham. Ze to je encrypted oznacuje ze je hodnota zasifrovana, tym ze su tam aj nezasifrovane hodnoty/variables(values) tak mi pride ok to takto prefixovat.